data/shaders/skybox_vs.glsl

```diff
@@ -0,0 +1,21 @@
+#version 150 core
+#include globals
+
+out vec3 v_TexCoord;
+
+void main() {
+    vec2 pos = gl_VertexID == 0 ? vec2(-1.0, -1.0) :
```

Comment:
You could take advantage of the vertex attributes of `quad_pipe` PSO if you use it

Reply:
I decided to leave it as it is. Quad pso doesn't have vertex coordinates, it would be overcomplication.